recipes/uproc/build.sh

```diff
@@ -0,0 +1,3 @@
+./configure --prefix=$PREFIX --enable-mmap	
+make
+make install
```

Comment:
Is there a `make check` we can run here?

Reply:
There is a `make check` command, but it does not run any tests. So I suppose there wouldn't be any use in executing it.

(On a side note: I am not the developer of this application and my knowlegde in C and its build systems is quite limited in general. So feel free to correct me if I'm wrong.)